assembly/assembly.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+
+  <id>assembly</id>
+
+  <formats>
+    <format>zip</format>
+    <format>tar.gz</format>
+  </formats>
+
+  <includeBaseDirectory>false</includeBaseDirectory>
+
+  <dependencySets>
+    <dependencySet>
+      <outputDirectory>lib</outputDirectory>
+      <useProjectArtifact>false</useProjectArtifact>
+      <useTransitiveDependencies>false</useTransitiveDependencies>
+      <excludes>
+        <exclude>io.camunda:c7-data-migrator-distro</exclude>
+        <exclude>com.h2database:h2</exclude>
+      </excludes> 
+    </dependencySet>
+    <dependencySet>
+      <outputDirectory>userlib/</outputDirectory>
+      <includes>
+        <include>com.h2database:h2</include>
+      </includes>
+    </dependencySet>
+  </dependencySets>
+
+   <files>
+  <file>
+    <source>${project.build.directory}/dependency/c7-data-migrator-distro-${project.version}.jar</source>
+    <outputDirectory>lib</outputDirectory>
+    <destName>c7-data-migrator.jar</destName>
+    <fileMode>0644</fileMode>
+  </file>
+    <file>
+      <source>resources/application.yml</source>
+      <outputDirectory>configuration/</outputDirectory>
+    </file>
+    <file>
+      <source>resources/start.sh</source>
+      <filtered>true</filtered>
+      <outputDirectory />
+      <fileMode>0755</fileMode>
+    </file>
+    <file>
+      <source>resources/start.bat</source>
+      <filtered>true</filtered>
+      <outputDirectory />
+      <fileMode>0755</fileMode>
+    </file>
+    <file>
+      <source>resources/README.txt</source>
+      <filtered>true</filtered>
+      <outputDirectory></outputDirectory>
+      <fileMode>0755</fileMode>
+    </file>
+    <file>
+      <source>../CAMUNDA-LICENSE-1.0.txt</source>
```

Comment:
/suggest Can we just name this file `LICENSE.txt` to make it unambiguously what this is about?

Reply:
Good point. Updated https://github.com/camunda/c7-data-migrator/pull/15/commits/515744651a1c8327ee29f29c50649dbad9423201